docs/modules/ROOT/pages/contributors/release.adoc

```diff
@@ -81,6 +81,29 @@ NONE
 
 This code block is included in KubeArchive's Pull Request template.
 
+=== Writing Release Notes
+
+The Release Notes should comply with the following rules:
+
+* They should be clear for users that install and operate KubeArchive but that did
+not look at the source code.
+* They are capitalized and start with a verb in the present tense. Often the verb
+will be "Adds", "Fixes", "Changes".
+* They should not mention specific internal implementation details. For example do
+not mention method names. They can mention KubeArchive's components: Sink, API and
+Operator. They can mention external specific details, for example a configuration key.
+
+=== Examples of Release Notes
+
+* Adds pprof endpoints enabled by default.
+* Changes the default log returned from the first container to the one specified in
+the annotation `kubectl.kubernetes.io/default-container`.
+* Allows users to filter resources by label on the majority of API endpoints.
+* Adds documentation about how to support a new database.
+* Adds retries when resource updates are not processed correctly.
+* Fixes a bug where resource updates were always rejected, causing performance problems.
+* Improves API speed when requesting resources cluster-wide.
+
```

Comment:
I think it would be useful to also mention bad examples of release notes:

* Vague
* Using technical language
* With fillers that makes it longer than needed

Reply:
Working on it.

Reply:
I added a couple of examples.